src/main/java/io/opentelemetry/semconv/resource/attributes/ResourceAttributes.java

```diff
@@ -128,7 +128,7 @@ public final class ResourceAttributes {
    *       href="https://docs.microsoft.com/en-us/rest/api/resources/resources/get-by-id">Fully
    *       Qualified Resource ID</a> of the invoked function, <em>not</em> the function app, having
    *       the form {@code
-   *       /subscriptions/<SUBSCIPTION_GUID>/resourceGroups/<RG>/providers/Microsoft.Web/sites/<FUNCAPP>/functions/<FUNC>}.
+   *       /subscriptions/<SUBSCIPTION_GUID/resourceGroups/<RG/providers/Microsoft.Web/sites/<FUNCAPP/functions/<FUNC}.
```

Comment:
This looks like it got a little mangled along the way.

Reply:
Nice catch